.github/workflows/main.yml

```diff
@@ -20,7 +20,7 @@ env:
 
   # If you change this value, please change it in the following files as well:
   # /Dockerfile
-  GO_VERSION: 1.20.4
+  GO_VERSION: 1.21.4
```

Comment:
nit: Go to the latest version (`v1.21.10`) since there was a CVE? I guess it's only relevant for building the release binaries anyway, but probably still good to update here.